mydoc/mydoc_cmor3_api.md

```diff
@@ -426,6 +426,7 @@ Returns upon success:
 
 ### cmor_time_varying_grid_coordinate()
 
+**DEPRECATED**: `cmor_time_varying_grid_coordinate` is no longer supported as of CMOR 3.11 and will be removed in future releases. 
```

Comment:
"and will be removed in future releases." -> "and will be removed in a future release."